src/Xamarin.Android.Build.Tasks/Utilities/ResourceParser.cs

```diff
@@ -40,17 +40,7 @@ internal static string GetNestedTypeName (string name)
 
 		internal string GetResourceName (string type, string name, Dictionary<string, string> map)
```

Comment:
Can we remove this method and just call `ResourceIdentifier.GetResourceName (type, name, map, Log);` directly?